src/pages/workspaces/workspace/jobHistory/CallTable.js

```diff
@@ -346,6 +347,37 @@ const CallTable = ({
             },
             ...(isAzure // GCP workspaces do not have inputs/outputs/logs (only log file is at top level, linked in workflow dashboard)
               ? [
+                  {
+                    size: { basis: 200, grow: 1 },
+                    field: 'cost',
+                    headerRenderer: () =>
+                      h(Sortable, { sort, field: 'cost', onSort: setSort }, [
+                        'Approximate Cost',
+                        h(
+                          TooltipTrigger,
+                          {
+                            content:
+                              'This is a calculated list price and does not include disk cost, subworkflow costs, or any cloud account discounts.',
+                          },
+                          [icon('info-circle', { style: { marginLeft: '0.4rem', color: colors.accent(1) } })]
+                        ),
+                      ]),
+                    cellRenderer: ({ rowIndex }) => {
+                      const { vmCostUsd, taskStartTime, taskEndTime, executionStatus, callCaching, subWorkflowId } = filteredCallObjects[rowIndex];
+                      if (taskStartTime && vmCostUsd) {
+                        if (executionStatus === 'Running') {
+                          const cost = getTaskCost(vmCostUsd, taskStartTime);
+                          return div([span({ style: { fontStyle: 'italic' } }, ['In Progress - ']), `$${cost}`]);
+                        }
+                        const cost = getTaskCost(vmCostUsd, taskStartTime, taskEndTime);
+                        return div({}, [`$${cost}`]);
+                      }
+                      if (executionStatus === 'Failed' || callCaching?.hit === true || !_.isEmpty(subWorkflowId)) {
```

Comment:
Why don't we want to show cost for failed tasks?

Reply:
Here we should still be showing the cost for failed tasks. This is in the event that a task fails before we can get a start/vm cost from TES. Without it, tasks that are failed with no additional data remain in a `Fetching cost information` state